BDArmory/Guidances/MissileGuidance.cs

```diff
@@ -834,6 +838,23 @@ public static float getGLimit(MissileLauncher ml, float thrust, float gLim, floa
 
             float currAoA = maxAoA;
 
+            if (ml.vessel.InVacuum())
+            {
+                if (thrust < gLim)
+                {
+                    currAoA = 90f;
+                }
+                else
+                {
+                    // If we're in vacuum, only consider engine thrust
+                    currAoA = Mathf.Asin(gLim / thrust);
```

Comment:
`Mathf.Asin` returns the angle in radians, so this'll need a `Mathf.Rad2Deg` factor.